src/main/java/com/myaudiolibrary/web/model/Artist.java

```diff
@@ -0,0 +1,44 @@
+package com.myaudiolibrary.web.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@Table(name="artist")
+public class Artist {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "ArtistId")
+    private Integer id;
+    @Column(name = "Name")
+    private String name;
+    @OneToMany(cascade = CascadeType.ALL, mappedBy = "artist")
```

Comment:
Le `CascadeType.ALL` est effectivement ici utilisable et il faut bien ajouter le `orphanRemoval` afin que la suppression de l'album soit bien persistée

Plus d'infos [ici](https://openjpa.apache.org/builds/2.4.0/apache-openjpa/docs/jpa_overview_meta_field.html#jpa_overview_meta_cascade) et [là](https://stackoverflow.com/questions/306144/jpa-cascadetype-all-does-not-delete-orphans)